tests/api2/test_alert.py

```diff
@@ -0,0 +1,86 @@
+from time import sleep
+
+import pytest
+from pytest_dependency import depends
+
+from auto_config import pool_name
+from middlewared.test.integration.utils import call, ssh
+
+
+def get_alert_by_id(alert_id):
+    results = call("alert.list")
+    for alert in results:
+        if alert['id'] == alert_id:
+            return alert
+
+
+def alert_exists(alert_id):
+    return isinstance(get_alert_by_id(alert_id), dict)
```

Comment:
`get_alert_by_id` returns None if the alert isn't found, no need to have this extra `alert_exists` function.

The caller can very simply do:
```
while get_by_alert_id(alert_id) is None:
    sleep(1)